extension/src/experiments/model/decorationProvider.ts

```diff
@@ -0,0 +1,44 @@
+import { EventEmitter, FileDecoration, ThemeColor, Uri } from 'vscode'
```

Comment:
[F] I moved this decoration provided back next to the tree it is associated with. I did the same thing for the other provider. I will be moving `extension/src/fileSystem/tree.ts` => `extension/src/repository/model/tree.ts` in the next PR to match the pattern we use in the experiments folder.